lib/std/start.zig

```diff
@@ -217,7 +219,29 @@ fn EfiMain(handle: uefi.Handle, system_table: *uefi.tables.SystemTable) callconv
         uefi.Status => {
             return @intFromEnum(root.main());
         },
-        else => @compileError("expected return type of main to be 'void', 'noreturn', 'usize', or 'std.os.uefi.Status'"),
+        else => |T| if (@typeInfo(T) == .error_union) {
+            const result = root.main() catch |err| {
+                std.log.err("{s}", .{@errorName(err)});
+                if (@errorReturnTrace()) |trace| {
+                    std.debug.dumpStackTrace(trace.*);
+                }
+                std.time.sleep(5 * std.time.ns_per_s);
```

Comment:
This part of the code is based on what @truemedian did. I'm not sure either so CC'ing them for some information.

Reply:
Magic sleep here was a debugging aid to prevent the uefi firmware from clearing the screen before I could read the any output because once EfiMain returns it will go to the next boot loader. Not necessary at all, unless that window to eead the error is desired.

Reply:
Ok, I likely will remove it and it'll be up to the program to do that if they want to.